src/modules/pages/components/RouteLayouts/UserNavigation.css

```diff
@@ -96,7 +96,9 @@
 }
 
 .walletAddress {
-  padding: 0 8px;
+  display: flex;
+  align-items: center;
```

Comment:
setting this to flex broken the alignment again :(

Remove these two lines and we're all good.